source/php-libraries.txt

```diff
@@ -37,6 +37,10 @@ Stand-alone Libraries
   **mongodb** extension. It is schema-less and supports fast object hydration
   and persistence, which makes it well-suited for modeling aggregation results.
 
+- `Xenus <https://github.com/abellion/xenus>`_ is an elegant MongoDB ODM
+   that supports events, relationships, embedded documents, and more.
+   It works on PHP 7+ and has an integration to the Laravel framework.
```

Comment:
These two lines are indented an extra space, which might generate an RST warning. I think they should align with the backtick on the first line of the bullet.